packages/gdl-frontend/pages/books/translations.js

```diff
@@ -333,9 +336,97 @@ const CustomGrid = styled('div')`
   }
 `;
 
-class MyTranslationsPage extends React.Component<{
-  intl: intlShape
-}> {
+const CustomSearchBar = styled('div')`
+  display: flex;
+  flex-direction: column;
+  @media (min-width: ${TABLET_BREAKPOINT}px) {
+    flex-direction: row;
+    justify-content: space-between;
+  }
+`;
+
+const CustomSearchbarItem = styled('div')`
+  width: 100%;
+  padding-top: 5px;
+`;
+
+function sortOnTitle(dir) {
```

Comment:
Som jeg skjønner er `dir` i dette tilfellet `1` eller `-1`. Dvs du opprettet en ny tilstand for å vite hvordan du skal sortere. Du slipper dette om du hadde sendt med `sortDirection` = `'asc' | 'desc'` og sjekke heller på det istedenfor. Det gjør det også mer lesbart, fordi jeg måtte lete meg fram til hva `1` stod for og hva `-1` stod for.